mkdocs.yml

```diff
@@ -1,76 +1,79 @@
 # Project information
```

Comment:
Question: Does mike auto-rewrite the mkdocs.yml? I am not sure I like this :sweat_smile: